projects/swimlane/ngx-charts/src/lib/bar-chart/bar-horizontal-2d.component.ts

```diff
@@ -86,7 +87,7 @@ import { ViewDimensions, LegendPosition, ScaleType, LegendOptions } from '../com
             [gradient]="gradient"
             [tooltipDisabled]="tooltipDisabled"
             [tooltipTemplate]="tooltipTemplate"
-            [seriesName]="group.name"
+            [seriesName]="group.name.toString()"
```

Comment:
I guess when strict mode is enabled. Could we maybe loosely type the input instead of calling `toString` in the templates?

Reply:
`seriesName` was of type `string` where as `group.name` is of type `StringOrNumberOrDate` that's why `.toString()` was needed.

I've now adjusted the type of `seriesName` to `StringOrNumberOrDate`. This requires us to be careful when checking if `this.seriesName` is `truthy` in `series-horizontal.component.ts`, since now `this.seriesName` can be a `number` with the value `0` which is `falsy`.

I guess we want to execute the corresponding logic if `this.seriesName` is set (i.e. also when `0`). Therefore, I convert `this.seriesName` to string in the two places where we check if it is set (see 5abae2fc5281bfff079a8d2ac23a257790b7d27c).

I hope that is ok?